src/lib/modules/ping/ping.go

```diff
@@ -27,6 +27,14 @@ type TestInterval struct {
 	Duration int64
 }
 
+// Response is used to decode the tsdb while sending data in the API
```

Comment:
`response is used to decode the tsdb blocks to data points that supports JSON encoding.`

Please change others according to this as well.